modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -149,7 +149,7 @@ $ aws kms get-key-policy --key-id <key_id_or_arn> --policy-name default --output
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Installer-Role",
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Worker-Role",
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-ControlPlane-Role",
```

Comment:
**issue:** Please change all instances of `aws-account-id` to `aws_account_id`. According to our user replaceable values conventions, we must use an `_`.

PS: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference for the "System or software variable to be replaced by the user" convention.